core/src/mindustry/logic/LUnitControl.java

```diff
@@ -16,7 +16,7 @@
     mine("x", "y"),
     flag("value"),
     build("x", "y", "block", "rotation", "config"),
-    getBlock("x", "y", "type", "building"),
+    getBlock("x", "y", "floor", "type", "building"),
```

Comment:
maybe add `floor` to end to make it compatible with older versions?

Reply:
OK, I'll also change edit another file to make it compatible with older logics